src/main/resources/application.yaml

```diff
@@ -31,9 +31,9 @@ quarkus:
         always-include: true
 
 honeycomb:
-    enabled: false
-    dataset: test
-    write-key: test
+    enabled: true
+    dataset: gateway-preview
+    write-key: gateway-preview
```

Comment:
We don't need to edit this file. The real config is managed in ansible and honeycomb is enabled. See https://gitlab.cee.redhat.com/nos/nos-ansible-playbooks/-/blob/master/playbooks/newcastle/files/gateway-devel-master/application.yaml

Reply:
Great :) i didn't know that.